page_objects/login_page.js

```diff
@@ -5,27 +5,32 @@ let TextView = require("../elements/text.view")
 
 let BasePage = require ("../base/base.page")
 
-
-
 let emailLocator = by.css('#email');
 let passwordLocator = by.css('#passwd');
 let submitbtnLocator = by.css('#SubmitLogin');
 let textloginLocator = by.css('#login_form>.page-subheading');
+let errorpassword = by.css('.alert li')
 
 
-class LoginPage extends BasePage {
-
-   
-    
+class LoginPage extends BasePage {   
     async login (email,password) {
         await allure.createStep(`Logging in ${email} / ${password}`, async () => {
             await this.getEmailInput().sendKeys(email);
             await this.getPassInput().sendKeys(password);
             await this.getSubmButInput().click();
+        })()
+    }
+
+    async error () {
+        return await this.getErrorElement().getText();
+    
+    }   
+
+    async clearText () {
```

Comment:
Please use name which will tell what this method is doing, like `clearCredentials()` or `clearLoginInfo()` etc.